wire-library/wire-runtime-swift/src/main/swift/WriteBuffer.swift

```diff
@@ -98,7 +98,8 @@ final class WriteBuffer {
     }
 
     func remove(count: Int, at offset: Int) {
-        memmove(storage.advanced(by: offset), storage.advanced(by: offset + count), self.count - offset)
+        let moveCount = self.count - (offset + count)
+        memmove(storage.advanced(by: offset), storage.advanced(by: offset + count), moveCount)
```

Comment:
This isn't a functional change, is it?

Reply:
Yeah, it is. Previous value was `self.count - offset`. New value is `self.count - (offset + count)`.

Reply:
Ahh missed that. LGTM